specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-05-01-preview/examples/fileImports/CreateFileImport.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "api-version": "2022-05-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "fileImportId": "73e01a99-5cd7-4139-a149-9f2736ff2ab5"
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalIinsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/FileImports/73e01a99-5cd7-4139-a149-9f2736ff2ab5",
+        "name": "73e01a99-5cd7-4139-a149-9f2736ff2ab5",
+        "type": "Microsoft.SecurityInsights/FileImports",
+        "properties": {
+          "importFile": {
+            "fileName": "myFile.json",
+            "fileSize": 4653,
+            "fileFormat": "JSON",
+            "fileContentUri": "https://sentinelimportswus2.blob.core.windows.net/78c2e51a-3cd3-4ca0-a2d4-e7effb9a05fe/43967a5e-47a7-474e-afb8-2081e9b99ca1/fileName.json?skoid=40ca3ff4-ed1d-4c65-a409-8c6caff8a6d5&sktid=72f988bf-86f1-41af-91ab-2d7cd011db47&skt=2022-03-25T21%3A12%3A51Z&ske=2022-03-25T22%3A12%3A51Z&sks=b&skv=2020-10-02&sv=2020-08-04&st=2022-03-25T21%3A12%3A51Z&se=2022-03-25T22%3A12%3A51Z&sr=b&sp=c&sig=5n0D%2FERS6ZOQdfdO2adleeSVOM4b6mQeds%2FWYCGm9pU%3D",
+            "deleteStatus": "NotDeleted"
+          },
+          "state": "WaitingForUpload",
+          "contentType": "StixIndicator",
+          "ingestionMode": "IngestAnyValidRecords",
+          "totalRecordCount": null,
+          "validRecordCount": null,
+          "ingestedRecordCount": null,
+          "createdTimeUTC": "2022-04-04T20:05:59.847136Z",
+          "filesValidUntilTimeUTC": "2022-04-05T20:05:59.8471361Z",
+          "importValidUntilTimeUTC": "2022-05-04T20:05:59.8471366Z",
+          "source": "mySource"
```

Comment:
is your RP setting up all this information? I don't see it from the PUT API.

Reply:
some properties are set during the PUT call (for example "source", "fileName"), and the other ones (like "createdTimeUTC", "ContentUri" are calculated in the RP. 
the fields that are expected to be received in the PUT are not marked with readonly true.